data/lang/ui-core/en.json

```
@@ -1915,6 +1915,10 @@
     "description": "",
     "message": "Pay fine of {amount}"
   },
+  "PENDING": {
+    "description": "Status if the mission has not yet been fully completed",
+    "message": "Pending\nReturn"
```

```suggestion
    "message": "Pending Return"
```

Prefer to avoid including manual line-breaks in the translation string. Display code can wrap the text as needed to fit the area.